.github/workflows/publish.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: Publish on PyPI
+
+on:
+    push:
+        tags:
+        # Tags that start with the "v" prefix are published on PyPI.
+            - v*
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+
```

Comment:
Would we not also need a step that installs `g++` and `make`? Or does the `ubuntu-latest` come with these pre-installed?

Reply:
I think that `ubuntu-latest` has them. At least when I was trying this out, everything went without a problem.

In the worst case, we fix it in a follow-up PR.